python/cuml/cuml/linear_model/logistic_regression.pyx

```diff
@@ -190,6 +190,7 @@ class LogisticRegression(Base,
 
     class_weight = CumlArrayDescriptor(order='F')
     expl_spec_weights_ = CumlArrayDescriptor(order='F')
+    _n_iter_ = CumlArrayDescriptor()
```

Comment:
Using this to avoid having a fitted attribute that exists even before `fit` has been called.

Reply:
We can fix this an alternative way (it's on my todos), if we want the output type to coerce I'd just follow the existing pattern and define it as `n_iter_`.